pkg/executor/client/job.go

```diff
@@ -333,11 +308,14 @@ func (c *JobExecutor) updateResultsFromPod(ctx context.Context, pod corev1.Pod,
 		}
 	}()
 
-	// wait for pod
-	l.Debug("poll immediate waiting for pod")
+	// wait for pod to be loggable
 	if err = wait.PollImmediate(pollInterval, c.podStartTimeout, executor.IsPodLoggable(ctx, c.ClientSet, pod.Name, c.Namespace)); err != nil {
 		l.Errorw("waiting for pod started error", "error", err)
-	} else if err = wait.PollImmediate(pollInterval, pollTimeout, executor.IsPodReady(ctx, c.ClientSet, pod.Name, c.Namespace)); err != nil {
+	}
```

Comment:
the idea behind this was for checking start pod errors like broken image first and then wait for pod execution result with longer timeout